src/applications/disability-benefits/all-claims/tests/migrations/migrations.unit.spec.js

```diff
@@ -237,4 +238,75 @@ describe('526 v2 migrations', () => {
       expect(migratedData).to.deep.equal(getData(Object.values(list)));
     });
   });
+
+  describe('08-paper-sync', () => {
+    it('should not change returnUrl if user left off on the veteran info page', () => {
+      const savedData = {
+        formData: {},
+        metadata: {
+          returnUrl: '/veteran-information',
+        },
+      };
+      const migratedData = reorderHousingIllnessRemoveFdc(savedData);
+      expect(migratedData.metadata.returnUrl).to.equal('/veteran-information');
+    });
+  });
+
+  it('should not change returnUrl if user left off on the contact info page', () => {
```

Comment:
These remaining tests are not wrapped within the `describe('08-paper-sync')`?

Reply:
oops lol. will fix. good catch.